dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/Enrollment.java

```diff
@@ -79,25 +78,19 @@
     private String orgUnit;
 
     @JsonProperty
-    private String orgUnitName;
+    private String enrolledAt;
```

Comment:
Are we renaming orgUnitName to enrolledAt or removing orgUnitName?

We need orgUnitName for all the three of tracker objects - tracked entity, enrollment and event. Yes, we already have UID of org unit. But it is much more convenient to return name as well since it is this property that comes useful for UI. Othwerwise, clients need to make another round trip to fetch org unit names.

Reply:
No, thats just how the diff looks. I removed orgUnitName since it wasn't using the translated name. Which clients use this? our own clients should all have metadata fully loaded? is this for external clients?

Reply:
Let's anyways wait for *name additions until we start using this for the capture app, for 2.34 (or more likely 2.34.1) this is anyways used for data import only. So then we can gradually add some of these things back if needed.